README.md

```diff
@@ -2,61 +2,30 @@
 
```

Comment:
- [ ] The branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly consider using this in the future projects

Reply:
Changed the branch name.
Old branch name : homePage  
New branch name: created_index_css